service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerMidEventTest.java

```diff
@@ -431,7 +435,8 @@ void shouldAddCheckYourOrderDetailsWhenOrderTypeIsSubmitted() {
                 .map(child -> child.getValue().getParty().getFullName())
                 .collect(Collectors.joining("\n"));
 
-            Map<String, Object> documentMap = mapper.convertValue(uploadedOrder, new TypeReference<>() {});
+            Map<String, Object> documentMap = mapper.convertValue(uploadedOrder, new TypeReference<>() {
+            });
```

Comment:
was this intentional?

Reply:
I think auto align might have changed it.. Shall I revert?

Reply:
please

Reply:
done